tsconfig.json

```diff
@@ -3,18 +3,19 @@
     "compilerOptions": {
         "downlevelIteration": true,
         "sourceMap": false,
-        "target": "es5",
-        "module": "commonjs",
-        "moduleResolution": "node",
+        "target": "ES2020",
+        "module": "ESNext",
+        "moduleResolution": "Node",
         "importHelpers": true,
         "removeComments": true,
-        "noImplicitAny": true,
+        "noImplicitAny": false,
         "strictPropertyInitialization": true,
         "strictNullChecks": true,
         "esModuleInterop": true,
+        "skipLibCheck": false,
```

Comment:
Is this needed? Isn't the default already set to false?

Reply:
Set it to true, as there are currently some collateral issues when building the lib. However, lib works ok even outputing these errors